src/main/java/eu/dissco/orchestration/backend/service/SourceSystemService.java

```diff
@@ -98,12 +193,35 @@ public JsonApiWrapper updateSourceSystem(String id, SourceSystem sourceSystem, S
     var sourceSystemRecord = new SourceSystemRecord(id, currentSourceSystem.version() + 1, userId,
         Instant.now(), null, sourceSystem);
     repository.updateSourceSystem(sourceSystemRecord);
+    updateCronJob(sourceSystemRecord, currentSourceSystem);
```

Comment:
Fair point, I was doubting about this. However, when you do a minor change in, I don't know the description, it might not be necessary. You can now always trigger it separately as well through the run endpoint.

Reply:
Can we include this in the request? A user can check a box that says "re-run ingestion on update"? Then pass that boolean to this function. 

Can be a separate user story.

Reply:
:100: But in a new user story https://naturalis.atlassian.net/browse/DD-1031